wgengine/router/router_linux.go

```diff
@@ -39,6 +40,11 @@ const (
 	netfilterOn       = preftype.NetfilterOn
 )
 
+// busyboxVersionRE extracts the major, minor, and patch versions
+// from a busybox banner string.
+// See: https://regex101.com/r/XGoKtD/1 for analysis.
+var busyboxVersionRE = regexp.MustCompile("(?m)^BusyBox\\Wv([0-9]+)\\.([0-9]+)\\.([0-9]+)\\W")
```

Comment:
use raw string literals for regexps.

But we've generally tried to avoid the regexp package for binary size reasons because it brings in huge unicode tables for matching e.g. `\p{Greek}`. If you must, though. I guess it's Linux-only.

Reply:
I used it as its already used for DNS INI parsing, but i missed that code was conditional for windows. I can re-write it into string scanning/indexing/splitting methods if thats preferable? should be pretty straightforward.